laravel/app/Services/OrderItemCsvImportService.php

```diff
@@ -33,21 +41,28 @@ public static function orderItemCsvImport($request)
             // 1行目のヘッダーは取り込まない
             if ($row_count > 1)
```

Comment:
下記のように記述することでインデントを下げることが出来ます。
インデントが深いコードは、コードの複雑さにも繋がるので出来る限り浅いインデントになるような実装を心がけましょう。
```suggestion
            if ($row_count == 1) {
                continue;
            }
```